docs/installation/production/manual/configuration/advanced/customize.rst

```diff
@@ -8,86 +8,91 @@ Customize Portal Theme (Recommended)
-
-For more advanced customization, you may use the **Custom Styles** and **Custom Template** options to completely replace the Home Page or Apps Library page CSS and HTML.
-
-Custom Styles
-=============
-
-The following settings can be used to add additional CSS to the Home page, Apps Library page, and portal-wide. Access the settings using the **Site Settings > Custom Styles** links on the admin pages.
-
-====================== =================================================================================
-Setting                Description
-====================== =================================================================================
-Portal Base CSS        CSS or path to a CSS file to be loaded on every Tethys Portal page. The file must be located within a valid static or public directory, such as in a Tethys app, Tethys extension, or Django app.
-Home Page CSS          CSS or path to a CSS file to be loaded on the Home page. The file must be located within a valid static or public directory, such as in a Tethys app, Tethys extension, or Django app.
-Apps Library CSS       CSS or path to a CSS file to be loaded on the Apps Library page. The file must be located within a valid static or public directory, such as in a Tethys app, Tethys extension, or Django app.
-====================== =================================================================================
-
-Custom Templates
-================
-
-The following settings can be used to override the templates for the Home page and Apps Library page. Access the settings using the **Site Settings > Custom Templates** links on the admin pages.
-
-====================== =================================================================================
-Setting                Description
-====================== =================================================================================
-Home Page Template     Path to alternate Home page template (will replace Home page template entirely). The template must be located within a valid templates directory, such as in a Tethys app, Tethys extension, or Django app.
-Apps Library Template  Path to alternate Apps Library page template (will replace Apps Library page template entirely). The template must be located within a valid templates directory, such as in a Tethys app, Tethys extension, or Django app.
-====================== =================================================================================
+GENERAL_SETTINGS
+++++++++++++++++
+
+============================== ================================================================================
+Setting                        Description
+============================== ================================================================================
+SITE_TITLE                     Title of the web page that appears in browser tabs and bookmarks of the site. Default is "Tethys Portal".
```

Comment:
In the Admin pages, these settings do not appear as shown in this table (upper snake case), making this a little confusing how it is written. Two suggestions:

1. Add another column to the table so it is something like this:

```rst
============================== ============================== ================================================================================
Admin Setting                  Site Setting Command           Description
============================== ============================== ================================================================================
Site Title                     SITE_TITLE                     Title of the web page that appears in browser tabs and bookmarks of the site. Default is "Tethys Portal".

...
```

2. Migrate the names of the settings in the database so that we only use the upper snake case everywhere, including in the Admin pages.

Reply:
I don't think we should migrate the names of the settings in the DB. I like the more readable names as they are. 

I think the setting names in the `portal_config.yml` file were made all caps so they would follow the convention of the Django settings. Even if we changed them to to follow the title case of the DB settings, we would still need to have underscores. I think just having a table that shows the correlation would be good, but we may need to have different tables in different places.

For example in the Production Installation guide these settings are only discussed in terms of the Admin Pages, so we probably don't need to include the `portal_config` key. However, on the customization page it would be good be more complete.

Also, in the suggested table you have a column called `Site Setting Command`. That column would more accurately be `Portal Config Yaml Key`. The site setting command for that setting would actually be called like this:

```bash
tethys site --site-title "Title of Site"
```

We could include a `Site Setting Command` column in the table for completeness:

```
============================== ============================== ============================== ================================================================================
Admin Setting                  Portal Config Yaml Key         Site Setting Command           Description
============================== ============================== ============================== ================================================================================
Site Title                     SITE_TITLE                     --site-title                   Title of the web page that appears in browser tabs and bookmarks of the site. Default is "Tethys Portal".

...
```

@swainn what are your thoughts?

Reply:
I like your table showing the relationship between the admin settings, portal yaml key, and site setting option. What if instead of having the table in multiple places, we put it in one place and then place `ref` links to it in the other places it is needed?

Reply:
These tables would go in two places:

http://docs.tethysplatform.org/en/stable/installation/production/manual/configuration/advanced/customize.html

and 

http://docs.tethysplatform.org/en/stable/tethys_portal/configuration.html#site-settings

If we only had it in one place I would say it should go in the second place. Is that okay:

![image](https://user-images.githubusercontent.com/6454472/183985741-f4cbdd86-3192-48bd-b292-b1e0b8047609.png)

Reply:
Yes.

Reply:
@swainn It looks like a similar table is also in the docker reference: http://docs.tethysplatform.org/en/stable/installation/production/docker/tethys_docker_reference.html#tethys-portal-site-settings

Do you think I should I just reference the table here too, and just explain that the docker environment variable will be the same as the portal config key?

Reply:
Sounds good to me.